daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala

```diff
@@ -166,12 +272,14 @@ class EvaluationOrderTest extends AnyFreeSpec with Matchers with Inside {
           Array(SParty(alice), SParty(bob)),
           alice,
         )
-        inside(res) { case SResultError(SErrorDamlException(err)) =>
-          err shouldBe a[CreateEmptyContractKeyMaintainers]
-
+        inside(res) {
+          case Success(
+                Left(SErrorDamlException(IE.CreateEmptyContractKeyMaintainers(T, _, _)))
+              ) =>
         }
-        msgs shouldBe allTraces
+        msgs shouldBe createTraces
```

Comment:
maybe move those inside the `inside`? It seems a bit odd to use `inside` without anything in the pattern match.